SafeguardSessions/Src/Modules/Constants.pqm

```diff
@@ -1,4 +1,7 @@
 let
+    // Version check
+    Constants.Skip = "Skip",

```

Comment:
Wouldn't "Yes" and "No" be simpler? Since the parameter name is "Skip Version Check" it sounds a bit strange that the possible values repeat the skip. For me, "Yes"/"No would be more straightforward, but I'm curious about the others' opinion.

Reply:
Done.